docs/src/showcase/blackhole.md

```diff
@@ -70,7 +70,7 @@ Details can be found in  [Keith et al. 2021](https://arxiv.org/abs/2102.12695).
 
 #### Feel free to skip reading this setup code!
```

Comment:
@ArnoStrouwen is there a nice way to fold this?

Reply:
You can collapse it by default like this:
https://github.com/SciML/LinearSolve.jl/blob/main/docs/src/index.md#reproducibility
But there is currently no way to copy it without still expanding it.